_specs/ecip-1064.md

```diff
@@ -0,0 +1,135 @@
+---
+lang: en
+ecip: 1064
+title: Communicating with 3rd parties
+status: Draft
+type: Informational
```

Comment:
This probably should be "Meta". We don't have a "Informational" category at this moment.

```suggestion
type: Meta
```